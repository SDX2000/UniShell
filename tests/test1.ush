echo "================"
echo $SCRIPT_NAME
echo "================"

#This is a comment
set a b # set a to b
env