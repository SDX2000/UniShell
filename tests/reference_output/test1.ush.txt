Variables
=========
INIT_DIR=/home/sandeepd/source/tools/unishell/tests
SCRIPT_DIR=/home/sandeepd/source/tools/unishell/tests
SCRIPT_NAME=test1.ush
SCRIPT_PATH=/home/sandeepd/source/tools/unishell/tests/test1.ush
a=b
================
test1.ush
================
